minitorch/tensor_ops.py

```diff
@@ -71,7 +73,8 @@ def map(fn):
                should broadcast with `a`
 
     Returns:
-        :class:`TensorData` : new tensor data
+        function: A function that takes a tensor, applies `fn` to each cell,
```

Comment:
Upon digging deeper here, it seems like the `Args` docs mix the arguments for the higher-order function and the function it returns...would it make sense to move the docs for `a` and `out` to be sub-points of this doc line (and so-forth for the other higher order functions).